sdk/monitor/azquery/custom_client.go

```diff
@@ -46,7 +46,7 @@ func NewLogsClient(credential azcore.TokenCredential, options *LogsClientOptions
 	}
 
 	authPolicy := runtime.NewBearerTokenPolicy(credential, []string{c.Audience + "/.default"}, nil)
-	azcoreClient, err := azcore.NewClient("azquery.LogsClient", version, runtime.PipelineOptions{PerRetry: []policy.Policy{authPolicy}}, &options.ClientOptions)
+	azcoreClient, err := azcore.NewClient(moduleName, version, runtime.PipelineOptions{PerRetry: []policy.Policy{authPolicy}}, &options.ClientOptions)
```

Comment:
Is there a way to differentiate between the two clients when tracing? Know the documentation you sent earlier said it should be by package name, but I'm still curious.

Reply:
The spans will have names like `Client.Method`, with the `otel.library.name` set to the value of `moduleName` so one can see exactly where the call originated.

Reply:
That makes sense now! Thank you!